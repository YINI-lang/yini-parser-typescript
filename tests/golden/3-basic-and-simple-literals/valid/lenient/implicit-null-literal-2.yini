^ ImplicitNullInLenient
val1 = 300
val2 =	// (!) Missing null is OK in lenient mode.
val3 = "My name"
