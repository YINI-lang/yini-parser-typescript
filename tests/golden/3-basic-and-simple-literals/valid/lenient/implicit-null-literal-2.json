{
    "ImplicitNullInLenient": {
        "val1": 300,
        "val2": null,
        "val3": "My name"
    }
}
